MD-Verlet-Integrator/md_verlet.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import psi4
+import os
+
+psi4.core.set_output_file('md_output.dat',False)
+
+molec = psi4.geometry("""
+H 0 0 0
+H 0 0 1
+""")
+
+# Global Constants (Atomic Units conversion)
+fs_timeau = 41.34137314
+amu2au = 1822.8884850
+
+#MD Options
+timestep =  5                       # Time step for each iteration in time atomic units
+max_md_step = 10000                 # Number of MD iterations
+veloc0 = np.zeros((2,3))            # Numpy array (natoms,3) with inital velocities
+trajec = True                       # Boolean: Save all trajectories in a single xyz file (md_trajectories)  for visualization
+grad_method = 'hf/3-21g'            # QC method (with basis set) for energy and gradient
+opt    = False                      # Optimize geometry using F=ma
+
+# Molecular Dynamics Function
+def molec_dyn(timestep,max_md_step,veloc0,grad_method,trajec = True,opt=False):
```

Comment:
Not sure this has to be a function, as it's fairly short & straightforward already.  Maybe could have `veloc_verlet` and `md_trajectories` defined above, and just have this code block below.